<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compresor de Mangas S</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ico.png</normaloff>ico.png</iconset>
  </property>
  <widget class="QGroupBox" name="ConvertirGrupo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>531</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="covertirCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Comprimir imagenes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="ConvertirBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ratebar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string extracomment="ratio"/>
    </property>
    <property name="accessibleDescription">
     <string extracomment="Mientras más a la izquierda sea menor será el tamaño resultate, pero tendrá menor calidad de imagen."/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="criterio">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string extracomment="Criterio de selección inteligente"/>
    </property>
    <property name="accessibleDescription">
     <string extracomment="Seleccionará automáticamente cuando y cuanto comprimir cada archivo según una heurística inteligente."/>
    </property>
    <property name="text">
     <string>Por defecto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Criterio de compresión:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="OtrasGrupo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>531</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QCheckBox" name="duplicadosCheck">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar archivos duplicados</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbzCheck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformar de zip a cbz</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="zipCheck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprimir carpetas en el interior a zip</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ordenarCheck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenar los cbz en formato numérico</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="eliminarCheck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar carpetas residuales</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="duplicadosBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="zipBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="cbzBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="ordenarBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="residualesBar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="inteligenteRadio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>inteligente</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="brutoRadio">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>bruto</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="manga">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Carpeta Objetivo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Iniciar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Reiniciar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
